src/main/java/com/rtg/vcf/eval/CombinedRocFilter.java

```diff
@@ -0,0 +1,39 @@
+package com.rtg.vcf.eval;
+
+import com.rtg.vcf.VcfRecord;
+import com.rtg.vcf.header.VcfHeader;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CombinedRocFilter extends RocFilter {
+    private final ArrayList<RocFilter> mRocFilters = new ArrayList<>();
+
+    /**
+     * Create a CombinedRocFilter which incorporates multiple RocFilters into one filter
```

Comment:
A little more informative description.
```suggestion
     * Create a CombinedRocFilter which accepts only records that are accepted by all delegate RocFilters.
```

Reply:
Good improvement; added.